class-pinterest-for-woocommerce.php

```diff
@@ -615,6 +617,17 @@ public static function set_api_version( $version ) {
 			return update_option( PINTEREST_FOR_WOOCOMMERCE_PINTEREST_API_VERSION, $version );
 		}
 
+		/**
+		 * Get API version used by the plugin.
+		 *
+		 * @since x.x.x
+		 *
+		 * @return string The API version.
+		 */
+		public static function get_api_version() {
+			return get_option( PINTEREST_FOR_WOOCOMMERCE_PINTEREST_API_VERSION, '' );
```

Comment:
With reference to https://developer.wordpress.org/plugins/plugin-basics/uninstall-methods/, I'm thinking we should delete the option (via https://github.com/woocommerce/pinterest-for-woocommerce/blob/30897feb5ada1962a61f6e1c9f98021e50381580/uninstall.php) when we are uninstalling the plugin. We can do this in a separate PR.

Reply:
https://github.com/woocommerce/pinterest-for-woocommerce/issues/931